src/Elliptic/Systems/Poisson/Equations.hpp

```diff
@@ -128,17 +128,17 @@ template <size_t Dim>
 struct Sources<Dim, Geometry::FlatCartesian> {
   using argument_tags = tmpl::list<>;
   static void apply(
```

Comment:
[optional] same question about moving to cpp file (for curved and other files below as well)